src/sentry/auth/helper.py

```diff
@@ -997,3 +999,6 @@ def EnablePartnerSSO(provider_key, sentry_org, provider_config):
         event=audit_log.get_event_id("SSO_ENABLE"),
         data=provider_model.get_audit_log_data(),
     )
+
+
+CHANNEL_PROVIDER_MAP = {ChannelName.FLY_IO.value: FlyOAuth2Provider}
```

Comment:
Should this go in `src/sentry/auth/partnership_config.py` instead?

Reply:
Unfortunately it cannot or else there'd be a circular dependency 😅 

I opted to stick it here next to the EnableSSO method since we're [importing from here](https://github.com/getsentry/getsentry/blob/master/getsentry/web/channel_provisioning/process.py#L12) anyways